src/bitcoin-cli.cpp

```diff
@@ -887,7 +887,8 @@ static void ParseError(const UniValue& error, std::string& strPrint, int& nRet)
             strPrint += ("error message:\n" + err_msg.get_str());
         }
         if (err_code.isNum() && err_code.getInt<int>() == RPC_WALLET_NOT_SPECIFIED) {
-            strPrint += "\nTry adding \"-rpcwallet=<filename>\" option to bitcoin-cli command line.";
+            strPrint += " For the CLI, specify the \"-rpcwallet=<filename>\" option before the command";
```

Comment:
I'm not sure if this new message makes it more clear. Specifically, "For the CLI" seems like it might confuse users because there's nothing opposed to that in the message.

Reply:
I've updated the test with the full error message, also I've changed the text a bit:
 
`Multiple wallets are loaded. Please specify a wallet by requesting the RPC through the /wallet/<filename> URI path. Using "bitcoin-cli", add the "-rpcwallet=<filename>" option before the command (run "bitcoin-cli -h" for help or "bitcoin-cli listwallets" to see which wallets are currently loaded).`

Reply:
I've re-tweaked it a bit, added a `\n` as current implementation and just set the error message that comes from the `utils` in the test's constant:
```
Multiple wallets are loaded. Please specify a wallet to use by requesting the RPC through the /wallet/<filename> URI path.
Using "bitcoin-cli", add the "-rpcwallet=<filename>" option before the command.
Run "bitcoin-cli -h" for help or "bitcoin-cli listwallets" to see which wallets are currently loaded.
````
WALLET_NOT_SPECIFIED = 'Multiple wallets are loaded. Please select a wallet to use'`

Reply:
I've re-considered your [preference](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1614026697) [above](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1127282385) @jonatack.